Source/WebCore/platform/graphics/displaylists/DisplayListItem.h

```diff
@@ -208,14 +208,19 @@ struct ApplyItemResult {
     std::optional<RenderingResourceIdentifier> resourceIdentifier;
 };
 
+enum class ApplyItemOptions : uint8_t {
+    Default,
+    FlushImagesAndWaitForCompletion
+};
```

Comment:
I think this should be used as `OptionSet<ApplyItemOption>`.

```
enum class ApplyItemOption : uint8_t {
    FlushImagesAndWaitForCompletion = 1 << 0,
};
```

Reply:
Indeed, much nicer -- didn't think of `OptionSet`, d'oh!